test/benchee/benchmark/runner_test.exs

```diff
@@ -282,228 +301,322 @@ defmodule Benchee.Benchmark.RunnerTest do
         ]
       }
 
-      %Suite{scenarios: [scenario]} = suite
-                                 |> test_suite
-                                 |> Benchmark.measure(TestPrinter)
+      %Suite{scenarios: [scenario]} =
+        suite
+        |> test_suite
+        |> Benchmark.measure(TestPrinter)
 
       # our previous run time isn't there anymore
       refute Enum.member?(scenario.run_times, 1_000_000)
     end
 
     test "global hooks triggers" do
       me = self()
+
       %Suite{
         configuration: %{
           warmup: 0,
           time: 100,
-          before_each: fn(input) -> send(me, :before); input end,
-          after_each: fn(_) -> send(me, :after) end
+          before_each: fn input ->
+            send(me, :before)
+            input
+          end,
+          after_each: fn _ -> send(me, :after) end
         }
       }
       |> test_suite
-      |> Benchmark.benchmark("job", fn -> :timer.sleep 1 end)
+      |> Benchmark.benchmark("job", fn -> :timer.sleep(1) end)
       |> Benchmark.measure(TestPrinter)
 
-      assert_received_exactly [:before, :after]
+      assert_received_exactly([:before, :after])
     end
 
     test "scenario hooks triggers" do
       me = self()
+
       %Suite{
         configuration: %{
           warmup: 0,
           time: 100
         }
       }
       |> test_suite
-      |> Benchmark.benchmark("job", {
-           fn -> :timer.sleep 1 end,
-           before_each: fn(input) -> send(me, :before); input end,
-           after_each: fn(_) -> send(me, :after) end,
-           before_scenario: fn(input) -> send(me, :before_scenario); input end,
-           after_scenario: fn(_) -> send(me, :after_scenario) end})
+      |> Benchmark.benchmark(
+        "job",
+        {fn -> :timer.sleep(1) end,
+         before_each: fn input ->
+           send(me, :before)
+           input
```

Comment:
does it complain about the `;` ? I know it's not great but I'd say the version above is a ton more readable than this here :(

Reply:
Yep - all functions with more than one clause go on multiple lines as of right now. I haven't seen anyone propose allowing `;`, though, so maybe that's something they'd consider.

Reply:
I don't wanna be the guy known for wanting to allow that. I agree that in 99% of the cases it's bad but here with the whole alignment it seemed nice